docs/licensing/chibios.md

```diff
@@ -3,42 +3,42 @@
 
 **About this document**
 
-This document provides a general overview on the licensing options available when using [ChibiOS](http://chibios.org) as **nanoFramework** RTOS component.
-We've been in touch with the ChibiOS sales team to clarify this as much as possible.
+This document provides a general overview of the licensing options available when using [ChibiOS](http://chibios.org) as the **nanoFramework** RTOS component.
+The nanoFramework team have been in contact with the ChibiOS sales team to ensure accuracy of the following information at time of print (October 2017).
 
-For details and to discuss your particular situation we strongly recommend getting in touch with the ChibiOS sales [here](http://chibios.org/dokuwiki/doku.php?id=chibios:licensing:quote).
+For details or to discuss your particular situation, we strongly recommend getting in touch with the ChibiOS [sales team](http://chibios.org/dokuwiki/doku.php?id=chibios:licensing:quote).
 
 
-## What exactly is being used from ChibiOS in **nanoFramework**?
+## What exactly is being used from ChibiOS in **nanoFramework firmware**?
```

Comment:
The double asterisk around nanoFramework is on purpose and used throughout the repos to highlight the name. So firmware should be left out.

Reply:
sorry for the delay, I will update this...

Reply:
Please do! 😉